js/packages/glimmer-idc/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules
```

Comment:
Question: since we're versioning all the JS at the top level, as opposed to per package, does this `.gitignore` file do anything? I only noticed because right now, we can't ignore `dist/` because of Drupal

Reply:
I think the packages end up getting some dependencies installed there. They have node_modules dirs at least. Not sure why that it is, but it would be pretty nasty to check all that in.

Reply:
i just wasn't sure how the `.gitignore` file here effected the top level repo. I don't think it actually does, though. I imagine this was autogen boilerplate